data/entities.js

```diff
@@ -6992,13 +6993,6 @@ module.exports = [
     categories: ['ad'],
     domains: ['*.liftdna.com'],
   },
-  {
-    name: 'Opentag',
-    company: 'Qubit',
-    categories: ['tag-manager'],
-    domains: ['*.qutics.com', 'd3c3cq33003psk.cloudfront.net'],
```

Comment:
qutics.com seems to have disappeared, add it back to Qubit?

Reply:
Done :heavy_check_mark: